src/stage1/generated.jl

```diff
@@ -313,6 +313,7 @@ function (::∂⃖{N})(::typeof(Base.getindex), a::Array, inds...) where {N}
     getindex(a, inds...), let
         EvenOddOdd{1, c_order(N)}(
             (@Base.aggressive_constprop Δ->begin
+                Δ isa ZeroTangent && return (NoTangent(), NoTangent(), map(x->NoTangent(), inds)...)
```

Comment:
I think this might be better?
```suggestion
                Δ isa AbstractZero && return (NoTangent(), Δ, map(Returns(Δ), inds)...)
```